src/main/kotlin/com/terraformation/backend/customer/api/UsersController.kt

```diff
@@ -13,6 +13,7 @@ import com.terraformation.backend.db.default_schema.UserId
 import io.swagger.v3.oas.annotations.Operation
 import io.swagger.v3.oas.annotations.media.Schema
 import java.time.ZoneId
+import java.util.*
```

Comment:
.Locale sufficient .. and in other new import locations?

Reply:
Got it, sounds good to leave it as-is for now.

Reply:
Just filed SW-2789 for this so I don't forget.